config/robots.rb

```diff
@@ -3264,6 +3264,46 @@
       }
     )
 
+    custom_robot(
+      'beckman-rvi-lib-pcr-xp-to-rvi-lib-prepool',
+      name: 'Beckman RVI Lib PCR XP => RVI Lib PrePool',
+      beds: {
+        bed(2).barcode => {
+          purpose: 'RVI Lib PCR XP',
+          states: %w[passed],
+          child: bed(4).barcode,
+          label: 'Bed 2'
+        },
+        bed(5).barcode => {
+          purpose: 'RVI Lib PCR XP',
+          states: %w[passed],
+          child: bed(4).barcode,
+          label: 'Bed 5'
+        },
+        bed(3).barcode => {
+          purpose: 'RVI Lib PCR XP',
+          states: %w[passed],
+          child: bed(4).barcode,
+          label: 'Bed 3'
+        },
+        bed(6).barcode => {
+          purpose: 'RVI Lib PCR XP',
+          states: %w[passed],
+          child: bed(4).barcode,
+          label: 'Bed 6'
+        },
+        bed(4).barcode => {
+          purpose: 'RVI Lib PrePool',
```

Comment:
Yes since the original story there has been an added step between RVI Lib PCR XP and RVI Hyb because of the logistics in limber.  RVI Lib PrePool is the intermediary step.